### DISCUSSION TITLE
Linux版本的opengnb启动，家里光猫会卡死，不知该从哪里排查问题？

### DISCUSSION THREAD

**Discussion by XduDSJ** [opengnb/opengnb#43] [Q&A]
运营商送的光猫，光猫拨号，未使用路由器，ZXHN F7615 MV30，有2.5G口接NAS，版本v1.5.0c
    1、armbian系统运行gnb，启动就会光猫卡死，ping丢包严重；
    2、绿联NAS，ssh运行gnb，启动后光猫就卡死；
    3、MacOS下的gnb一直在运行，无影响。
这个该从哪里排查问题？

还想问个问题，想在nas上用docker部署gnb，不知是否可行？

**Comment by gnbdev** [comment]
用tcpdump 抓包检查是否存在大量的由gnb发出的地址探测包。加微信 26253，拉你进技术支持群发自我的 iPhone在 2024年11月18日，12:11，XduDSJ ***@***.***> 写道：﻿
运营商送的光猫，光猫拨号，未使用路由器，ZXHN F7615 MV30，有2.5G口接NAS。
1、armbian系统运行gnb，启动就会光猫卡死，ping丢包严重；
2、绿联NAS，ssh运行gnb，启动后光猫就卡死；
3、MacOS下的gnb一直在运行，无影响。
这个该从哪里排查问题？
还想问个问题，想在nas上用docker部署gnb，不知是否可行？

**Reply by XduDSJ** [reply]
冒昧问一下，这个微信号怎么加？我搜不到呢。

**Comment by gnbdev** [comment]
忘记开权限了，现在可以添加了。发自我的 iPhone在 2024年11月18日，14:58，XduDSJ ***@***.***> 写道：﻿
冒昧问一下，这个微信号怎么加？我搜不到呢。
